### PR TITLE
Contribute "Ari's Garden"

### PR DESCRIPTION
**Pull request by teesloane**
Hi there!

I thought I would propose contributing a recipe site I have built in Elm. It was well received on the elm subreddit so thought it might fit in here, too. Cheers!

### CODE REVIEW

**Review by nelsonic — COMMENTED** [review]
@teesloane Great project! Good looking healthy recipes. Thanks for sharing. 🎉 
Possibly worth also linking to the website where it's deployed: https://arisgarden.theiceshelf.com ?
image

<hr />

The _reason_ the Travis-CI build for this PR is failing is that one of the links in the `README.md` is `404` ... 😞 

See: https://travis-ci.org/sporto/awesome-elm/builds/645487077#L486
image

https://pbs.twimg.com/profile_images/443794371586977792/NxKUNpOQ.jpeg is `404` 🤷‍♂ 
image

A quick search for `NxKUNpOQ.jpeg` in the codebase:
https://github.com/sporto/awesome-elm/search?utf8=%E2%9C%93&q=NxKUNpOQ.jpeg
image
Reveals that this is the Twitter Profile pic for Evan https://twitter.com/czaplic (_who recently updated his profile pic on Twitter..._) hence the `404`...
https://twitter.com/czaplic
image

## Potential Solution:

We could _either_ update the `` src to the latest profile picture URL:
https://twitter.com/czaplic/photo > https://pbs.twimg.com/profile_images/1217159920749096962/FHEpqhXA_400x400.jpg
image

***Or*** link to his GitHub profile pic:
https://avatars3.githubusercontent.com/u/1658058
Which is based on his GitHub user ID and will _never_ `404` (_unless he deletes his GitHub account_)

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 * [Elm Playground](https://ccamel.github.io/playground-elm/index.html) - Pure SPA (with routing) exploring various aspects of Elm.
 * [Elm Hacker News PWA](https://github.com/elmariofredo/elm-hn-pwa) - A progressive web app built with Elm version 0.18, using the official Hacker-News API
 * [Elm Narrative Engine](https://github.com/jschomay/elm-narrative-engine) - A framework for building interactive fiction style stories in Elm. Detailed example [elmnarrativeengine.com](http://elmnarrativeengine.com) shows how to build a "chose your own adventure" game; perfect for beginners. **Elm v.0.19**
-* [Ari's Garden](https://github.com/theiceshelf/arisgarden) - A recipe site built as an SPA that also makes use of the Elm parser.
+* [Ari's Garden](https://github.com/theiceshelf/arisgarden) - A recipe [site](https://arisgarden.theiceshelf.com/) built as an SPA that also makes use of the Elm parser.
```
Comment (nelsonic) [inline README.md:66]: 👍

File: README.md [hunk 2]
```diff
@@ -432,7 +432,7 @@ Here are some notable ones:
...
 
-<a href="https://twitter.com/elmlang" target="_blank"><img src="./assets/elm-logo.svg" height="100" width="100" /></a> | <a href="https://twitter.com/czaplic" target="_blank"><img src="https://pbs.twimg.com/profile_images/443794371586977792/NxKUNpOQ.jpeg" width="100" /></a> | <a href="https://twitter.com/rtfeldman" target="_blank"><img src="https://pbs.twimg.com/profile_images/635812303342956545/Fo4RyEgH.jpg" width="100" /></a> | <a href="https://twitter.com/elmweekly" target="_blank"><img src="https://s3.amazonaws.com/revue/profiles/images/000/046/579/thumb/elm_weekly_logo.png" width="100" /></a>
+<a href="https://twitter.com/elmlang" target="_blank"><img src="./assets/elm-logo.svg" height="100" width="100" /></a> | <a href="https://twitter.com/czaplic" target="_blank"><img src="https://avatars3.githubusercontent.com/u/1658058" width="100" /></a> | <a href="https://twitter.com/rtfeldman" target="_blank"><img src="https://pbs.twimg.com/profile_images/635812303342956545/Fo4RyEgH.jpg" width="100" /></a> | <a href="https://twitter.com/elmweekly" target="_blank"><img src="https://s3.amazonaws.com/revue/profiles/images/000/046/579/thumb/elm_weekly_logo.png" width="100" /></a>
```
Comment (nelsonic) [inline README.md:435]: Fixed `404` on `` `src` for Evan's profile pic. ✅ 

Before:  https://github.com/sporto/awesome-elm#who-to-follow
evan-profile-pic-broken

After: https://github.com/sporto/awesome-elm/blob/96238423b4d99a8475f5cb0909156f2b0b9365bb/README.md#who-to-follow
evan-profile-pic-fixed

**Review by nelsonic — APPROVED** [review]
Great addition @teesloane 🙌 
Thanks for fixing the `404` ✅

**Comment by teesloane** [comment]
No problem! Happy to contribute 🙌

**Comment by sporto** [comment]
Great, thanks

**Pull request merged**